const-oid/Cargo.toml

```diff
@@ -16,12 +16,16 @@ readme = "README.md"
 edition = "2021"
 rust-version = "1.57"
 
+[dependencies]
+arbitrary = { version = "1.2.0", features = ["derive"], optional = true }
+
 [dev-dependencies]
 hex-literal = "0.3"
 
 [features]
-std = []
+arbitrary = ["std", "dep:arbitrary"]
```

Comment:
This requires an MSRV bump, which we're not ready to do yet.

I plan on landing #757 first.

Reply:
It's feasible to have the same semantic by changing the feature name and using the implicit `arbitrary` feature inside. However, with [this](https://github.com/RustCrypto/formats/pull/761/files#diff-4256d0986a772eb1e13583d5ddc3d60712c86ebf05dbd03b57a52329d0f17cc0R33), the removal of the `?` will unconditionally enable the `const-oid` dependency when the feature is enabled, and I don't know an other way of doing this.

Reply:
We bumped MSRV to 1.60 so this is fine now